### PR TITLE
Update TODO List and vscode settings

### PR DESCRIPTION
**Pull request by liyone**

### CODE REVIEW

**Comment by liyone** [comment]
go over it once please :)

**Review by IndiW — APPROVED** [review]
lgtm

File: node_modules/yarn/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  <a href="https://circleci.com/gh/yarnpkg/yarn"><img alt="Circle Status" src="https://circleci.com/gh/yarnpkg/yarn.svg?style=shield&circle-token=5f0a78473b0f440afb218bf2b82323cc6b3cb43f"></a>
+  <a href="https://ci.appveyor.com/project/kittens/yarn/branch/master"><img alt="Appveyor Status" src="https://ci.appveyor.com/api/projects/status/0xdv8chwe2kmk463?svg=true"></a>
+  <a href="https://dev.azure.com/yarnpkg/yarn/_build"><img alt="Azure Pipelines status" src="https://dev.azure.com/yarnpkg/yarn/_apis/build/status/Yarn%20Acceptance%20Tests"></a>
+  <a href="https://discord.gg/yarnpkg"><img alt="Discord Chat" src="https://img.shields.io/discord/226791405589233664.svg"></a>
+  <a href="http://commitizen.github.io/cz-cli/"><img alt="Commitizen friendly" src="https://img.shields.io/badge/commitizen-friendly-brightgreen.svg"></a>
+</p>
+
+---
+
+**Fast:** Yarn caches every package it has downloaded, so it never needs to download the same package again. It also does almost everything concurrently to maximize resource utilization. This means even faster installs.
```
Comment (IndiW) [inline node_modules/yarn/README.md]: wait why is this here?

Comment (IndiW) [reply node_modules/yarn/README.md]: this file*

Comment (liyone) [reply node_modules/yarn/README.md]: good catch ty, no idea
